doc/source/changes/version_0_25_1.rst.inc

```diff
@@ -37,3 +37,18 @@ Fixes
 
     >>> arr['a2:'] = arr[':a1'].set_labels('a', 'a2,a3')
     >>> arr['a2:'] = arr[':a1'].drop_labels('a')
+
+* fixed getting a subset of an array by mixing boolean filters and other filters (closes :issue:`246`):
+
+  >>> arr = ndtest((4, 6)).set_labels('b', '0..5')
```

Comment:
* indentation is wrong (should be 4)
* I would use ndrange('a=0..2,b=0..3') and x.b < 2 to make the test shorter